experimental/addons/gateway-traits/definitions/http-route.cue

```diff
@@ -27,13 +27,13 @@ template: {
 					if parameter.gatewayName == _|_ {
 						name: "traefik-gateway"
 					}
-					namespace: "vela-system"
-					if parameter.listenerName != _|_ {
```

Comment:
Why removed the listener name?

Reply:
![1661415198511](https://user-images.githubusercontent.com/75570810/186612319-156d2c2d-b780-468b-8be7-8da5ab4f1b3a.jpg)
I add it, it can be matched with the port.